projects/1.20/assets/maturi-delight/maturidelight/lang/zh_cn.json

```
@@ -0,0 +1,21 @@
+{
+    "item.maturidelight.suspicious_item": "这个章鱼烧上放了什么东西……",
+    "item.maturidelight.takoyaki_raw": "生章鱼烧",
+    "item.maturidelight.takoyaki": "章鱼烧",
+    "item.maturidelight.takoyaki_soysauce": "淋有酱油的章鱼烧",
+    "item.maturidelight.okonomiyaki_raw": "生什锦烧",
+    "item.maturidelight.okonomiyaki": "什锦烧",
+    "item.maturidelight.okonomiyaki_soysauce": "淋有酱油的什锦烧",
+    "item.maturidelight.dough_raw": "生面团",
+    "block.maturidelight.takoyaki_maker": "章鱼烧器",
+    "block.maturidelight.plate": "铁板",
+    "itemGroup.maturidelight": "祭祀乐事",
+    "advancements.maturidelight.root.title": "祭祀乐事",
+    "advancements.maturidelight.root.description": "认识祭祀活动的食物！",
```

```suggestion
    "advancements.maturidelight.root.description": "来一起了解祭典上的食物吧！",
```
经典的`～学ぼう`系，常见于各类教程视频标题